keyboards/matchstickworks/normiepad/info.json

```diff
@@ -0,0 +1,50 @@
+{
+    "keyboard_name": "NormiePad",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP2", "GP4", "GP11", "GP12"],
+        "rows": ["GP13", "GP29", "GP28", "GP20", "GP19", "GP18"]
+    },
+    "processor": "RP2040",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0003",
+        "vid": "0x4D57"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
THis appears to match the numpad_6x4 layout, and the config here should be changed to match: 
https://github.com/qmk/qmk_firmware/tree/master/layouts/default/numpad_6x4

Reply:
Do you mean just changing "LAYOUT" to "LAYOUT_numpad_6x4" (plus the keymap.c rename), or is there something more? Adding the ASCII graphic to keymap.c? I'm unsure what this ask is specifically.

Reply:
Specifically, to change the name, tweak the layout to match the community layout, and: 
```json
    "community_layouts": [ "numpad_6x4" ],
```

Reply:
Mine appears to be wired up a little differently than the default 6x4. The `Plus` and `Enter` are on a row higher than the default, won't that cause issues?

I did notice a mistake with my `h` and `w` tags on info.json being wrong (copied over from my other design) so at least I'll be able to fix that.

Reply:
@drashna Bumping my question

Reply:
Sorry.  Yes and no.  You'd want to move the positions in the info.json to match the community layout.  However, how they are wired up doesn't actually matter to the info.json.  As long as the correct matrix positions are assigned, it will work correctly.   You'd need to alter the keymap.c files, though. 

But my point is that you should use the community layout, whenever possible.  It allows users to maintain one keymap that is usable on different hardware and makes setting things up for them easier.

Reply:
Ah, okay! Thanks for the explainer! I'll keep that in mind for future boards.